microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -361,4 +361,18 @@ For the `TopoLVM image`, see link:https://catalog.redhat.com/software/containers
 [id="microshift-4-14-4-bug-fixes"]
 ==== Bug fixes
 
-* Previously, a backup was created every time the {microshift-short} service started. These backups caused system backups to fail. Now, automated backups only occur on full system starts. (link:https://issues.redhat.com/browse/OCPBUGS-23076[*OCPBUGS-23076*])
\ No newline at end of file
+* Previously, a backup was created every time the {microshift-short} service started. These backups caused system backups to fail. Now, automated backups only occur on full system starts. (link:https://issues.redhat.com/browse/OCPBUGS-23076[*OCPBUGS-23076*])
+
+[id="microshift-4-14-5-dp"]
+=== RHBA-2023:7601 - {microshift-short} 4.14.5 bug fix update advisory
+
+Issued: 2023-12-05
+
+{product-title} release 4.14.5, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:7601[RHBA-2023:7601] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:7599[RHSA-2023:7599] advisory.
+
+For the `TopoLVM image`, see link:https://catalog.redhat.com/software/containers/lvms4/topolvm-rhel9/63972de3adcb55263891b983?container-tabs=dockerfile[lvms4/topolvm-rhel9].
+
+[id="microshift-4-14-5-bug-fixes"]
+==== Bug fixes
+
+* Previously, when MicroShift was installed on a non-ostree system, an error was logged during greenboot health check stating that the ostree directory did not exist. MicroShift still started normally. Now, the system properly logs the ostree status as “not an ostree system” and an error message is not produced.
```

Comment:
I'm guessing based on line 367 that these MicroShift mentions should be attributes:
```suggestion
* Previously, when {microshift-short} was installed on a non-ostree system, an error was logged during greenboot health check stating that the ostree directory did not exist. {microshift-short} still started normally. Now, the system properly logs the ostree status as “not an ostree system” and an error message is not produced.
```

Reply:
Whoops, great catch!